app/assets/stylesheets/moj/_govuk_overrides.scss

```diff
@@ -169,3 +169,60 @@ td {
 .data-request #data_request_location:disabled {
   border-color: $grey-2;
 }
+
+.govuk-warning-text__assistive {
```

Comment:
What do you think of grouping this CSS in a separate file under `/stylesheets/moj/components` and then including it in `/stylesheets/application.css.scss`?